README.md

```diff
@@ -236,6 +237,8 @@ const offer = await seaport.createBuyOrder({
   startAmount: 1.2,
 })
 ```
+#### Offer Limits
+Total value of buy orders must not exceed 100 x wallet balance.
```

Comment:
The total value of all of your outstanding orders cannot exceed 100 times your wallet's balance.